scripts/gh-projects/examples/matchline/additions/add-plan-refinements.sh

```
@@ -0,0 +1,67 @@
+# shellcheck source=../../../lib.sh
+source "$SCRIPT_DIR/../../../lib.sh"
+
+# Existing parents
+P0_NUM=5
+P1_NUM=15
+
+# -----------------------------------------------------------------------------
+# Phase 0 additions
+# -----------------------------------------------------------------------------
+F=$(prep_body "$SCRIPT_DIR/p0-cost-tracker.md" "$P0_NUM")
+read P0_COST_URL P0_COST_NUM _ <<<"$(create_child \
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)